connectors/file/translator-parquet/src/test/java/org/teiid/translator/parquet/TestParquetExecution.java

```diff
@@ -0,0 +1,207 @@
+                "	id integer ,\n" +
+                "	\"month\" string ,\n" +
+                "	name string ,\n" +
+                "	\"year\" long ,\n" +
+                "	CONSTRAINT PK0 PRIMARY KEY(id)\n" +
+                ") OPTIONS (\"teiid_parquet:FILE\" 'dir', \"teiid_parquet:PARTITIONED_COLUMNS\" 'year,month');";
+
+        VirtualFileConnection connection = new JavaVirtualFileConnection(UnitTestUtil.getTestDataPath());
+
+        ArrayList results = helpExecute(ddl, connection, "select name from Table1 WHERE \"year\"=2019 and \"month\"='January' and name='Michael'");
+        Assert.assertEquals("[[Michael]]",results.toString());
+    }
+
+    @Test
+    public void testParquetExecutionWithDirectoryBasedPartitioningAndPartitionedColumnSelect() throws Exception {
+        String ddl = "CREATE FOREIGN TABLE Table1 (\n" +
+                "	id integer ,\n" +
+                "	\"month\" string ,\n" +
+                "	name string ,\n" +
+                "	\"year\" long ,\n" +
+                "	CONSTRAINT PK0 PRIMARY KEY(id)\n" +
+                ") OPTIONS (\"teiid_parquet:FILE\" 'dir', \"teiid_parquet:PARTITIONED_COLUMNS\" 'year,month');";
+
+        VirtualFileConnection connection = new JavaVirtualFileConnection(UnitTestUtil.getTestDataPath());
+
+        ArrayList results = helpExecute(ddl, connection, "select name, \"year\" from Table1 WHERE \"year\"=2019 and \"month\"='January'");
+        Assert.assertEquals("[[Michael, 2019], [Anne, 2019]]",results.toString());
+    }
+
+    @Test(expected = TranslatorException.class)
+    public void testParquetExecutionWithIllegalOperatorInPredicate() throws Exception {
+        String ddl = "CREATE FOREIGN TABLE Table1 (\n" +
+                "	contacts integer[] ,\n" +
+                "	id integer ,\n" +
+                "	last string ,\n" +
+                "	name string ,\n" +
+                "	CONSTRAINT PK0 PRIMARY KEY(id)\n" +
+                ") OPTIONS (\"teiid_parquet:FILE\" 'people.parquet');";
+
+        VirtualFileConnection connection = new JavaVirtualFileConnection(UnitTestUtil.getTestDataPath());
+
+        ArrayList results = helpExecute(ddl, connection, "select * from Table1 WHERE id<>24");
+    }
+
+    @Test(expected = TranslatorException.class)
+    public void testParquetExecutionWithNonListType() throws Exception {
+        String ddl = "CREATE FOREIGN TABLE Table1 (\n" +
+                "	id integer ,\n" +
+                "	last string ,\n" +
+                "	name string ,\n" +
+                "	userPreferences string,\n" +
+                "	CONSTRAINT PK0 PRIMARY KEY(id)\n" +
+                ") OPTIONS (\"teiid_parquet:FILE\" 'MapLogicalType.parquet');";
+
+        VirtualFileConnection connection = new JavaVirtualFileConnection(UnitTestUtil.getTestDataPath());
+
+        ArrayList results = helpExecute(ddl, connection, "select * from Table1");
+
+    }
+
+    @Test
+    public void testParquetExecutionWithRowFilterEq() throws Exception {
```

Comment:
@shawkins this test is failing despite the correct rowGroupFilter, please have a look! It is not returning any row.